File: java/UberBlack.java
package POO.java;

import java.util.ArrayList;
import java.util.Map;

public class UberBlack extends Car {
    Map<String, Map<String,Integer>> typeCarAccepted;
    ArrayList<String> seatMaterial;
    
    public UberBlack(String placa, Account driver, Map<String, Map<String,Integer>> typeCarAccepted, ArrayList<String> seatMaterial){
        super(placa, driver);
        this.typeCarAccepted = typeCarAccepted;
        this.seatMaterial = seatMaterial;
    }
}

File: php/uberBlack.php
<?php
require_once('car.php');
class UberBlack extends Car {
    public $typeCarAccepted;
    public $seatMaterial;

    public function __construct($placa, $driver, $typeCarAccepted, $seatMaterial){
        parent::__construct($placa, $driver);
        $this->typeCarAccepted = $typeCarAccepted;
        $this->seatMaterial = $seatMaterial;
    }
}
?>